src/PerformanceTests/Common/CheckGCServer.cs

```diff
@@ -14,7 +14,12 @@ class CheckGCServer : IProfile, INeedPermutation
 
     public void Configure(Configuration cfg)
     {
-        if (Permutation.GarbageCollector == GarbageCollector.Client && GCSettings.IsServerGC) throw new InvalidOperationException("GarbageCollector is set to Server but must be Client.");
-        if (Permutation.GarbageCollector == GarbageCollector.Server && !GCSettings.IsServerGC) throw new InvalidOperationException("GarbageCollector is set to Client but must be Server.");
+        if (Permutation.Platform == Platform.NetFramework)
+        {
+            if (Permutation.GarbageCollector == GarbageCollector.Client && GCSettings.IsServerGC) throw new InvalidOperationException("GarbageCollector is set to Server but must be Client.");
+            if (Permutation.GarbageCollector == GarbageCollector.Server && !GCSettings.IsServerGC) throw new InvalidOperationException("GarbageCollector is set to Client but must be Server.");
+        }
+        
+        // TODO: GC cannot only be configured via csproj in .net core. Should we throw an exception if the GC mode doesn't match?
```

Comment:
@ramonsmits are you ok if we let this in here for now?

Reply:
@danielmarbach With .net framework GC was also only configurable via the config. The test runner patches the XML for it.

Reply:
Ok then I can fix it

Reply:
I would write a log entry with level WARN to identify that this setting is ignored with .net core if we cannot patch the config as is done for .net framework.

Reply:
Scratch that comment. Is patches the App.Config but we would need to patch the csproj which does not sound feasible?

Reply:
yeah, the configuration happens in the csproj file, that's not patchable. It might be possible to patch the the `runtimeconfig.json` file (but not with the current XML patching obviously).

Reply:
I only found this snippet

Reply:
Also reading something about setting `complus_gcserver` env var, probably that could be done too for example in the batch file and also in the runner but patching that json file with just a string.Replace seems pretty trivial.

Reply:
>All settings in this group, except for the "System.GC.Server" property, are placed into a file called runtimeconfig.template.json

https://docs.microsoft.com/en-us/dotnet/core/tools/project-json-to-csproj

😭